assets/training/distillation/components/data_generation/spec.yaml

```diff
@@ -83,8 +83,19 @@ inputs:
 
   enable_chain_of_thought:
     type: string
-    default: "true"
+    default: "false"
```

Comment:
What is the reason to change this to false by default?

Reply:
This is not required for the conversational task, so changed the default value to false.

Reply:
Keep it as it is. We want users to Use COT.

Reply:
@vikasagrawal-ms 
FYI, we won't need it for conversational, so made the default to false. 
Will revert.